Tools/LyTestTools/ly_test_tools/environment/process_utils.py

```diff
@@ -12,8 +12,8 @@
 import subprocess
 import ctypes
 
+import ly_test_tools
 import ly_test_tools.environment.waiter as waiter
-from ly_test_tools import WINDOWS, MAC
```

Comment:
Shouldn't this be from ly_test_tools import LINUX, MAC, WINDOWS to match the file above? Why two different patterns?

Reply:
The pattern should be made consistent across tools, to import ly_test_tools and inspect its members, not copy their values. I'm trying to make the pattern consistent but would not be surprised that there are still extraneous from-imports